addon/index.js

```diff
@@ -84,6 +104,14 @@ function updatePrefs() {
     TESTPILOT_PREFIX: env.TESTPILOT_PREFIX
   });
 
+  // wait about 5 minutes before prompting user for survey
```

Comment:
Might want to toss this along with `getRandomExperiment` and `showRandomSurvey` into the survey module and call this as `survey.init()` or something, so that all the survey-related stuff stays corralled in that module